src/node.cc

```diff
@@ -4956,6 +4956,15 @@ int Start(int argc, char** argv) {
 #if ENABLE_TTD_NODE
   bool chk_debug_enabled = debug_options.inspector_enabled();
 
+  std::string envDoRecordVar;
+  bool envDoRecord = SafeGetenv("DO_TTD_RECORD", &envDoRecordVar) &&
```

Comment:
>DO_TTD_RECORD [](start = 33, length = 13)

does the environment variable per process and child processes only or do we need to reset it somewhere?